<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNarcissisticNumbers</class>
 <widget class="QDialog" name="CNarcissisticNumbers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator - T162</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/resources/calc.png</normaloff>:/resources/calc.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QSpinBox" name="base">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>36</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Number of Values per Thread:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="numThreads">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Base (Between 2 and 32):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="maxRange">
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="numPerThread">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="byRange">
     <property name="text">
      <string>Range of Numbers:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of Threads:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="minRange">
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>to</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="byNumbers">
     <property name="text">
      <string>List of Numbers:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="numList"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
